makefiles/unzip.mk

```diff
@@ -34,12 +36,11 @@ endif
 unzip-package: unzip-stage
 	# unzip.mk Package Structure
 	rm -rf $(BUILD_DIST)/unzip
-	mkdir -p $(BUILD_DIST)/unzip
+	mkdir -p $(BUILD_DIST)/unzip/$(MEMO_PREFIX)$(MEMO_SUB_PREFIX)/share
 
 	# unzip.mk Prep unzip
-	cp -a $(BUILD_STAGE)/unzip/$(MEMO_PREFIX)$(MEMO_SUB_PREFIX) $(BUILD_DIST)/unzip
-	mkdir -p $(BUILD_DIST)/unzip/$(MEMO_PREFIX)$(MEMO_SUB_PREFIX)/share
-	mv $(BUILD_DIST)/unzip/$(MEMO_PREFIX)$(MEMO_SUB_PREFIX)/man $(BUILD_DIST)/unzip/$(MEMO_PREFIX)$(MEMO_SUB_PREFIX)/share
+	cp -a $(BUILD_STAGE)/unzip/$(MEMO_PREFIX)$(MEMO_SUB_PREFIX)/bin $(BUILD_DIST)/unzip/$(MEMO_PREFIX)$(MEMO_SUB_PREFIX)
+	cp -a $(BUILD_STAGE)/unzip/$(MEMO_PREFIX)$(MEMO_SUB_PREFIX)/man $(BUILD_DIST)/unzip/$(MEMO_PREFIX)$(MEMO_SUB_PREFIX)/share
```

Comment:
Why can't we just `cp -a $(BUILD_STAGE)/unzip $(BUILD_DIST)`?

Reply:
`cp -a $(BUILD_STAGE)/unzip/$(MEMO_PREFIX)$(MEMO_SUB_PREFIX)/man $(BUILD_DIST)/unzip/$(MEMO_PREFIX)$(MEMO_SUB_PREFIX)/share`

you will get `/usr/man` instead of `/usr/share/man`.

Reply:
Can't you specify the manpage prefix? If not, then this is fine; most projects allow this, but not all, so; you can set it up when builiding.